src/validations/role/index.ts

```diff
@@ -0,0 +1,19 @@
+export default async function validate(
+  author: string,
+  space,
+  proposal,
+  options
+): Promise<boolean> {
+  const onlyMembers = options.onlyMembers || space.filters?.onlyMembers;
+  const members = (space.members || []).map((address) => address.toLowerCase());
+  const strategies = [...space.strategies];
+  const getRoleIndex = strategies.findIndex(
+    ({ name }) => name === 'contract-call'
+  );
+  if (members.includes(author.toLowerCase())) return true;
+  if (onlyMembers) return false;
+  if (getRoleIndex <= 2) {
+    return false;
+  }
```

Comment:
the role is based on strategy name? doesn't make sense to me 🤔 or am I looking it wrong?